code/modules/mob/living/basic/space_fauna/spider/young_spider/young_spider_subtypes.dm

```diff
@@ -130,13 +145,73 @@
 		complete_text = "%SOURCE%'s wounds mend together.",\
 	)
 
+	AddElement(/datum/element/web_walker, /datum/movespeed_modifier/average_web)
+
 /// Prevent you from healing other tangle spiders, or healing when on fire
 /mob/living/basic/spider/growing/young/tangle/proc/can_mend(mob/living/source, mob/living/target)
 	if (on_fire)
 		balloon_alert(src, "on fire!")
 		return FALSE
 	return TRUE
 
+
+/// Will differentiate into the "tank" giant spider.
+/mob/living/basic/spider/growing/young/tank
+	grow_as = /mob/living/basic/spider/giant/tank
+	name = "young tank spider"
+	desc = "Furry and purple, it looks defenseless. This one has dim yellow eyes."
+	icon = 'icons/mob/simple/arachnoid.dmi'
+	icon_state = "young_tank"
+	icon_dead = "young_tank_dead"
+	maxHealth = 50
+	health = 50
+	damage_coeff = list(BRUTE = 0.5, BURN = 0.5, TOX = 0.5, CLONE = 0.5, STAMINA = 0.5, OXY = 1)
+	melee_damage_lower = 10
+	melee_damage_upper = 15
+	speed = 1
+
+/mob/living/basic/spider/growing/young/tank/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/healing_touch,\
+		heal_brute = 5,\
+		heal_burn = 5,\
+		heal_time = 2 SECONDS,\
+		self_targetting = HEALING_TOUCH_SELF_ONLY,\
+		interaction_key = DOAFTER_SOURCE_SPIDER,\
+		valid_targets_typecache = typecacheof(list(/mob/living/basic/spider/growing/young/tank, /mob/living/basic/spider/giant/tank)),\
```

Comment:
Indiana bones said why I want tank spiders to be able to heal while in combat instead of them having to leave off combat for a while to heal I made it include both types because current tangle spider does and maybe it does need it